spec.md

```diff
@@ -31,9 +31,21 @@ sequence of operations the program encapsulates.
 - `"format"`: Signifies the utilization of the PHIR/JSON format.
 - `"version"`: Represents the semantic version number adhering to the PHIR spec.
 - `"metadata"`: An optional segment where users can incorporate additional details. This segment holds potential for
-future expansion, possibly to guid compilation processes and error modeling.
+future expansion, possibly to guide compilation processes and error modeling.
 - `"ops": [{...}, ...]`: A linear sequence denoting the operations and blocks that constitute the program.
 
+## Metadata
+
+The metadata field takes several parameters that can be used to specify information relevant to the entire program
+
+### Metadata Options
+
+| parameter               | options                 | description                                                     |
+| ----------------------- | ----------------------- | --------------------------------------------------------------- |
+| `"default_time_unit"`   | `"s", "ms", "us", "ns"` | The default time unit in the "duration" section of qop metadata |
+| `"default_angle_unit"`  | `"rad", "pi"`           | The default angle unit in the "angles" section of a qop         |
```

Comment:
"default_" is not needed as a prefix. The second could be:
```suggestion
| `"angular_unit"`  | `"rad", "pi"`           | The default angle unit in the "angles" section of a qop         |
```

But also if we are adding these two, we will need to revise the rest of the spec. I suggest not introducing these two units in this PR to keep the focus of this PR toward parallelism only.